README.md

```diff
@@ -37,7 +37,7 @@ At this moment, the scripts available are:
 
 Currently, you can play with our subgraph here:
 
-- [Ropsten](https://thegraph.com/explorer/subgraph/zero-collateral/zero-collateral).
+- [Ropsten](https://thegraph.com/explorer/subgraph/teller-protocol/teller-protocol).
```

Comment:
You should add production mainnet url
Also you should use the urls you provided (so we can change/switch between the environments)
```
The subgraph for mainnet was already deployed.
TheGraph URLs (website):
Ropsten: https://thegraph.com/explorer/subgraph/salazarguille/teller-protocol-subgraph-ropsten
Rinkeby: https://thegraph.com/explorer/subgraph/salazarguille/teller-protocol-subgraph-rinkeby
Mainnet: https://thegraph.com/explorer/subgraph/teller-protocol/subgraph-mainnet
```